docs/rule/kotlin_library.soy

```diff
@@ -69,10 +69,109 @@ the <code>resources</code> argument.
 {/call}
 
 {call buck.arg}
-  {param name: 'extra_arguments' /}
+  {param name: 'free_compiler_args' /}
```

Comment:
Why not `compiler_args`? Even `extra_compiler_args`. It's not what "free" means here and we use `extra` in some other rules.

Reply:
I was naming these to match the naming semantics that the kotlin tooling itself uses for consistency